landing-page/src/components/partials/Footer.vue

```diff
@@ -10,8 +10,7 @@
           </nav>
         </div>
         <div class="col-6 text-right">
-          <p><a href="#">O aplikaci</a> | <a href="#">Data</a> | <a href="#">Licence</a> | <a href="#">Kontakt</a></p>
-          <p><a href="#">Přihlásit se</a></p>
+          <p><a href="http://www.cityvizor.cz/#login" target="_blank">Přihlásit se</a></p>
```

Comment:
Tohle mi přijde divné. Je to rozcestník do různých instancí CityVizoru, ale login je do jedné z nich. Je to tak dobře?

Reply:
Konzultoval jsem to s Pavlou Kadlecovou a vec se ma nasledovne:
* administrace (login) je spolecny pro vsechny instance CV, tzn. toto je OK
* po prihlaseni se nic nestane -> pridal jsem tam automaticky redirect na profily
* vzhledem k nedostatecne osetrenym pravum (uz je v PB) dochazi k tomu, ze spravce jednoho konkretniho profilu porad v administraci vidi vsechny ostatni profily, i kdyz by nemel. Nastesti je tohle pouze kosmeticka vada a tyto profily tento spravce nemuze upravovat.